src/figuresClasses.ts

```diff
@@ -1,11 +1,79 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: string = 'triangle';
+
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
```

Comment:
The constructor parameters 'a', 'b', and 'c' are being manually assigned to class properties. With TypeScript, you can use the access modifier 'public' to automatically create and assign properties, which removes the need for manual assignment.